software/sw_build.mk

```
@@ -63,14 +64,23 @@ TEMPLATE_LDS=src/$@.lds
 
 IOB_SOC_OPENCRYPTOLINUX_CFLAGS ?=-Os -nostdlib -march=rv32imac -mabi=ilp32 --specs=nano.specs -Wcast-align=strict
 
-IOB_SOC_OPENCRYPTOLINUX_INCLUDES=-I. -Isrc 
+IOB_SOC_OPENCRYPTOLINUX_INCLUDES=-I. -Isrc -Isrc/crypto/McEliece -Isrc/crypto/McEliece/common
 
 IOB_SOC_OPENCRYPTOLINUX_LFLAGS=-Wl,-Bstatic,-T,$(TEMPLATE_LDS),--strip-debug
 
 # FIRMWARE SOURCES
 IOB_SOC_OPENCRYPTOLINUX_FW_SRC=src/iob_soc_opencryptolinux_firmware.S
 IOB_SOC_OPENCRYPTOLINUX_FW_SRC+=src/iob_soc_opencryptolinux_firmware.c
 IOB_SOC_OPENCRYPTOLINUX_FW_SRC+=src/printf.c
+
+# NOTE(Ruben): To speed up simulation, the following files can be commented out to greatly reduce filesize from around ~120Kb to ~20Kb.
+
+IOB_SOC_OPENCRYPTOLINUX_FW_SRC+=src/versat_crypto.c
+IOB_SOC_OPENCRYPTOLINUX_FW_SRC+=src/versat_crypto_tests.c
+IOB_SOC_OPENCRYPTOLINUX_FW_SRC+=src/crypto/aes.c
+IOB_SOC_OPENCRYPTOLINUX_FW_SRC+=$(wildcard src/crypto/McEliece/*.c)
+IOB_SOC_OPENCRYPTOLINUX_FW_SRC+=$(wildcard src/crypto/McEliece/common/*.c)
```

Removed comment, and added conditional expression to include sources only for the FPGA:
```suggestion
ifneq ($(SIMULATION),1)
IOB_SOC_OPENCRYPTOLINUX_FW_SRC+=src/versat_crypto.c
IOB_SOC_OPENCRYPTOLINUX_FW_SRC+=src/versat_crypto_tests.c
IOB_SOC_OPENCRYPTOLINUX_FW_SRC+=src/crypto/aes.c
IOB_SOC_OPENCRYPTOLINUX_FW_SRC+=$(wildcard src/crypto/McEliece/*.c)
IOB_SOC_OPENCRYPTOLINUX_FW_SRC+=$(wildcard src/crypto/McEliece/common/*.c)
endif
```